forge/src/main/java/com/teamresourceful/resourcefulbees/common/block/FakeFlower.java

```diff
@@ -0,0 +1,65 @@
+package com.teamresourceful.resourcefulbees.common.block;
+
+import com.teamresourceful.resourcefulbees.common.block.base.RenderingBaseEntityBlock;
+import com.teamresourceful.resourcefulbees.common.blockentity.FakeFlowerEntity;
+import com.teamresourceful.resourcefulbees.common.blockentity.SolidificationChamberBlockEntity;
+import com.teamresourceful.resourcefulbees.common.blockentity.centrifuge.CentrifugeBlockEntity;
+import com.teamresourceful.resourcefulbees.common.registry.minecraft.ModItems;
+import com.teamresourceful.resourcefulbees.common.utils.FluidUtils;
+import com.teamresourceful.resourcefulbees.common.utils.ModUtils;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.BaseEntityBlock;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import net.minecraft.world.phys.shapes.BooleanOp;
+import net.minecraft.world.phys.shapes.CollisionContext;
+import net.minecraft.world.phys.shapes.Shapes;
+import net.minecraft.world.phys.shapes.VoxelShape;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.stream.Stream;
+
+public class FakeFlower extends RenderingBaseEntityBlock {
+
+    private static final VoxelShape FULL_SHAPE = Stream.of(
+            Block.box(0, 0, 0, 16, 2, 16)
```

Comment:
This should just be `Block.box(...)` no need for stream reduce, also for future reference to or multiple shapes use `Shapes.or(...)`